h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var Controller = require('../base/controller');
+
+class LozengeController extends Controller {
```

Comment:
I got some feedback on this from Rob and SeanR: It's fine for `SearchBarController` to pass in the parent element to `LozengeController`, though less consistent with the other controllers it is a more convenient API from the calling code's point of view. So you can leave this the way it is if you want, but maybe just rename the `el` argument to something like `containerEl` so it's clear that it's the containing element rather than the lozenge element itself?